src/dedupe-hub/api/app/matching/matcher/exact/exact_matcher.py

```diff
@@ -24,12 +31,18 @@ def process(
     ) -> Dict[str, List[MatchDTO]]:
         self.set_start()
 
+        log.info('[Matching] Exact match processing started!')
+        log.info(f'[Matching] Messy data: {messy}')
```

Comment:
do you think it's good to print all messy data?

Reply:
@killev messy is "data that user upload" here.

Reply:
So I think to see what come into Exact Matcher and then see what coming out from there. The same for Gazzetteer Matcher. @killev